ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/StaticQueryExecutor.java

```diff
@@ -119,6 +119,15 @@ public static void validate(
   ) {
     final Query queryStmt = statement.getStatement();
 
+    if (!statement.getConfig().getBoolean(KsqlConfig.KSQL_PULL_QUERIES_ENABLE_CONFIG)) {
+      throw new KsqlRestException(
+          Errors.badStatement(
+              "Pull queries are disabled on this KSQL server - please set "
+                  + KsqlConfig.KSQL_PULL_QUERIES_ENABLE_CONFIG + "=true to enable this feature. "
+                  + "If you intended to issue a push query, resubmit the query with EMIT CHANGES.",
```

Comment:
reword to "resubmit with EMIT CHANGES syntax" or "resubmit with EMIT CHANGES clause" ?

Reply:
changed to `EMIT CHANGES clause`